cmd/livepeer_bench/livepeer_bench.go

```diff
@@ -151,8 +163,22 @@ func main() {
 		time.Sleep(300 * time.Millisecond)
 	}
 	wg.Wait()
-	fmt.Fprintf(os.Stderr, "Took %0.4v seconds to transcode %v segments of total duration %vs (%v concurrent sessions)\n",
-		time.Since(start).Seconds(), segCount, segTotalDur, *concurrentSessions)
+	statsTable := tablewriter.NewWriter(os.Stderr)
+	stats := [][]string{
+		{"Concurrent Sessions", fmt.Sprintf("%v", *concurrentSessions)},
+		{"Total Segs Transcoded", fmt.Sprintf("%v", segCount)},
+		{"Real-Time Segs Transcoded", fmt.Sprintf("%v", realTimeSegCount)},
+		{"* Real-Time Segs Ratio *", fmt.Sprintf("%0.4v", float64(realTimeSegCount)/float64(segCount))},
```

Comment:
Curious why asterisks are added here? Anything particularly special about this stat relative to the other ones?

Reply:
Wanted to highlight this and `Real-Time Duration Ratio` in some manner - as a user I would personally be most interested in quickly eyeballing these two ratios. WDYT?

Reply:
Got it. Sounds reasonable.